cassovary-core/src/main/scala/com/twitter/cassovary/util/FastUtilUtils.scala

```diff
@@ -61,4 +61,6 @@ object FastUtilUtils {
 
   def newInt2IntOpenHashMap(): mutable.Map[Int, Int] =
```

Comment:
@szymonm in your suggestion the inner mutable.IndexedSeq will have unboxing/boxing, yes?